stdlib/public/Concurrency/TaskGroup+addTask.swift.gyb

```diff
@@ -31,7 +31,7 @@ import Swift
 %          'ThrowingDiscardingTaskGroup'
 %        ],
 %        [
-%          '@available(SwiftStdlib 6.2, *)',
+%          '@available(SwiftStdlib 5.1, *)',
```

Comment:
small win, but it's nice to fix just a single line rather than hunt around 8 locations where to fix it